CONTRIBUTE

```diff
@@ -0,0 +1,69 @@
+CONTRIBUTING TO TQDM
+=====================
+
+This file describes how to contribute changes to the project, and how to upload to the pypi repository.
+
+HOW TO COMMIT YOUR CONTRIBUTIONS
+-----------------------------------------------------------
+
+Contributions to the project is made using the `Fork & Pull` model. Here's a quickstart on how to do that (your mileage may vary depending on your git config):
+
+- first create an account on github.
+- then go on the tqdm page `https://github.com/tqdm/tqdm` and fork it (click on the fork button).
+- now make a local clone of the project: `git clone https://github.com/your_account/tqdm.git`
+- do your changes on your local copy.
+- commit your changes `git commit -m "my message"`
+- TEST YOUR CHANGES (see below).
+- push to your github account `git push origin`
+- and now you can make a `Pull Request` from your github fork by going to the webpage and clicking on `Pull Request`. You can then add a message to describe your proposal.
+- Your changes will then be reviewed and integrated into the project if OK.
+
+TESTING
+-------------
+
+Before submitting a Pull Request, you need to make sure that all the unit tests pass. This will ensure that your changes did not introduce any regression in the features that already working.
+
+To run the test, cd to the root of the tqdm folder (in the same folder as this file), and then type the following (you need to install the `nosetest` python module before):
+
+```
+nosetests --with-coverage --cover-package=tqdm -v tqdm/
+python -m flake8 tqdm/_tqdm.py
+```
```

Comment:
I usually use the `Makefile` with `make test` `make coverage` and `make flake8`.